lib/uplinkc/testdata/object_test.c

```diff
@@ -138,6 +153,15 @@ void handle_project(ProjectRef project) {
         free_list_objects(&objects_list);
     }
 
+    { // Delete objects
```

Comment:
Imo, the delete test should verify that the object is no longer in existence after the delete. Either by doing `get` and expecting an error or by doing a `list` and expecting that the object is not in the list

Reply:
Absolutely, there is a [todo in the test](https://github.com/bryanchriswhite/libstorj/blob/V3/test/tests.c#L381) for this. Maybe we should track that in the review process for the libstorj code?

Reply:
ok. Could you explain how this section of the test (`lib/uplinkc/testdata/object_test.c`) relates to `libstorj/test/tests.c`, specifically `check_delete_file()`, the function you linked above?

Reply:
I wasn't paying attention before when you first commented; this is a C test for libuplink and the tests.c file in libstorj is for testing the libstorj api, they are unrelated.

Reply:
7d1ddc182

Reply:
:roll_eyes: before you say anything... adding another commit to free that thing

Reply:
18e507129